CODEOWNERS

```diff
@@ -48,6 +48,8 @@ extensions/filters/common/original_src @snowp @klarose
 /*/extensions/tracers/datadog @cgilmour @palazzem @mattklein123
 # tracers.xray extension
 /*/extensions/tracers/xray @marcomagdy @lavignes @mattklein123
+# tracers.skywalking extension
+/*/extensions/tracers/skywalking @dio @lizan @wbpcode
```

Comment:
Tagging this to confirm that @dio and @lizan are on-board as being the extension sponsor and CODEOWNERS.

Reply:
@htuch I confirm.